server/devpi_server/filestore.py

```diff
@@ -148,6 +144,7 @@ def __init__(self, msg, code=None, url=None):
 class FileEntry(object):
     BadGateway = BadGateway
     hash_spec = metaprop("hash_spec")  # e.g. "md5=120938012"
+    # BBB keep this for a while (1 year?) for older devpi-web
```

Comment:
I do not understand the comment

Reply:
When removing the eggfragment metaprop all versions of devpi-web released so far will break. I'm uncertain whether that is an issue or not, so I left it in for now.